pyproject.toml

```diff
@@ -23,13 +23,39 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4"
-importlib-metadata = {version = "^4.0", python = "<3.8"}
+python = ">=3.9,<3.13"
```

Comment:
```suggestion
python = ">=3.9,<4"
```
Maybe better like this?

Reply:
![image](https://github.com/user-attachments/assets/8cf56d15-9e22-4c86-bd48-c2c10d1ab1c9)
There was a conflict when updating to version 4, so I had to limit it to version 3.13.